packages/plugins/test/mocha/cli/cli.spec.ts

```diff
@@ -0,0 +1,333 @@
+      expect(proposeSpy.calledWithMatch(expectedParams)).to.be.true
+    })
+
+    it('--networks mainnets', () => {
+      const args = ['propose', scriptPath, '--networks', 'mainnets']
+
+      makeCLI(args, sphinxContext)
+
+      const expectedParams = {
+        confirm: false,
+        isTestnet: false, // Changed to false for mainnets
+        isDryRun: false,
+        silent: false,
+        scriptPath,
+        sphinxContext: sinon.match.any,
+        targetContract: undefined,
+        forceRecompile: true,
+      }
+
+      expect(proposeSpy.calledWithMatch(expectedParams)).to.be.true
+    })
+
+    it('--networks mainnets --confirm', () => {
+      const args = [
+        'propose',
+        scriptPath,
+        '--networks',
+        'mainnets',
+        '--confirm',
+      ]
+
+      makeCLI(args, sphinxContext)
+
+      const expectedParams = {
+        confirm: true, // confirm is true
+        isTestnet: false,
+        isDryRun: false,
+        silent: false,
+        scriptPath,
+        sphinxContext: sinon.match.any,
+        targetContract: undefined,
+        forceRecompile: true,
+      }
+
+      expect(proposeSpy.calledWithMatch(expectedParams)).to.be.true
+    })
+
+    it('--networks mainnets --target-contract MyContract', () => {
+      const args = [
+        'propose',
+        scriptPath,
+        '--networks',
+        'mainnets',
+        '--target-contract',
+        'MyContract',
+      ]
+
+      makeCLI(args, sphinxContext)
+
+      const expectedParams = {
+        confirm: false,
+        isTestnet: false,
+        isDryRun: false,
+        silent: false,
+        scriptPath,
+        sphinxContext: sinon.match.any,
+        targetContract: 'MyContract', // Specified target contract
+        forceRecompile: true,
+      }
+
+      expect(proposeSpy.calledWithMatch(expectedParams)).to.be.true
+    })
+  })
+
+  describe('deploy', () => {
+    let deploySpy: sinon.SinonStub
+
+    beforeEach(() => {
+      deploySpy = sinon.stub(sphinxContext, 'deploy')
+    })
+
+    it('fails if no script path is included', async () => {
+      const args = ['deploy', '--network', 'testnet']
+
+      makeCLI(args, sphinxContext)
+
+      expect(exitSpy.calledWith(1)).to.be.true
+      expect(consoleErrorSpy.called).to.be.true
+      expect(consoleErrorSpy.firstCall.args[0]).to.include(
+        'Not enough non-option arguments: got 0, need at least 1'
+      )
+    })
+
+    it('fails if no --network is provided', () => {
+      const args = ['deploy', scriptPath]
+
+      makeCLI(args, sphinxContext)
+
+      expect(deploySpy.called).to.be.false
+      expect(consoleErrorSpy.called).to.be.true
+      expect(consoleErrorSpy.firstCall.args[0]).to.include(
+        'Missing required argument: network'
+      )
+    })
+
+    it('--network ethereum', () => {
+      const network = 'testnet'
+      const args = ['deploy', scriptPath, '--network', network]
```

Comment:
```suggestion
    it('--network ethereum', () => {
      const network = 'ethereum'
      const args = ['deploy', scriptPath, '--network', network]
```

Minor suggestion, there are also a number of other test cases here with similar minor errors. Not a huge deal, but it is kind of confusing for someone trying to understand the tests.

Reply:
Nice catch